src/runners/RunnerUtil.ml

```diff
@@ -106,6 +106,16 @@ let parse_cli () =
     ("-cf", Arg.Unit (fun () -> r_cf := true), "Run cashflow checker and print results.");
     ("-jsonerrors", Arg.Unit (fun () -> r_json_errors := true), "Print errors in JSON format");
   ] in 
+
+  let mandatory_usage = " -init init.json [-istate input_state.json]" ^
```

Comment:
This is incorrect for all executables other than `scilla-runner` (which uses `cli.ml` and which you have fixed there). The mandatory arguments here are, as was before, just `-libdir /path/to/stdlib input.scilla`.